packages/design-system-docs/src/scripts/components/ReactPropDoc.jsx

```diff
@@ -69,17 +69,38 @@ class ReactPropDoc extends React.PureComponent {
   }
 
   render() {
+    // Specify ARIA roles attribute for table to ensure responsive HTML table is accessible to screen readers
     return (
-      <tr>
-        <td>
+      <tr role="row">
+        <td role="cell" headers="columnname">
+          <span className="docs_table__column-header ds-u-font-weight--bold" aria-hidden="true">
```

Comment:
Down the road it would be nice to have this `aria-hidden` be dynamic, depending on if the text is hidden or not. This is good for now though!